src/redux/selectors/index.js

```diff
@@ -22,20 +22,20 @@ export const patientSelector = createSelector(
 export const supportedResourcesSelector = createSelector(
   [resourceIdsGroupedByTypeSelector],
   (resourceIdsGroupedByType) => Object.entries(resourceIdsGroupedByType)
-  // do not include Patient, Observation, or unknown/unsupported:
+    // do not include Patient, Observation, or unknown/unsupported:
     .filter(([resourceType]) => !!RESOURCE_TYPES[resourceType])
     // sort by label:
     .sort(([t1], [t2]) => ((RESOURCE_TYPES[t1] < RESOURCE_TYPES[t2]) ? -1 : 1))
-    .reduce((acc, [resourceType, resourceIds]) => ({
-      ...acc,
-      [resourceType]: resourceIds,
-    }), {}),
+    .reduce((acc, [resourceType, subtypes]) => acc.concat({
+      resourceType,
+      subtypes,
```

Comment:
looks like `subtypes` will be used in selector based on this, to select all patient resources and get dates...